idb/postgres/internal/writer/writer.go

```diff
@@ -57,9 +70,10 @@ var statements = map[string]string{
 		creator = EXCLUDED.creator, params = EXCLUDED.params, dao_name = EXCLUDED.dao_name,
 		asset_id = EXCLUDED.asset_id, deleted = FALSE`,
 	upsertAccountAppStmtName: `INSERT INTO account_app
-		(addr, app, localstate, deleted)
-		VALUES($1, $2, $3, FALSE) ON CONFLICT (addr, app) DO UPDATE SET
-		localstate = EXCLUDED.localstate, deleted = FALSE`,
+		(addr, app, localstate, voting_start, voting_end, deleted)
+		VALUES($1, $2, $3, $4, $5, FALSE) ON CONFLICT (addr, app) DO UPDATE SET
+		localstate = EXCLUDED.localstate, voting_start = EXCLUDED.voting_start,
+		voting_end = EXCLUDED.voting_end, deleted = FALSE`,
```

Comment:
why don't we set all app attributes as JSON in a new column?

BTW, is this inserting app record (new DAO), or a proposal?

Reply:
This is inserting a proposal. App attributes are first encoded and then stored in JSON column format. We are storing only the required attributes in other columns like voting_end, voting_start etc.